nbformat/v4/nbformat.v4.schema.json

```diff
@@ -351,16 +351,14 @@
             "mimebundle": {
                 "description": "A mime-type keyed dictionary of data",
                 "type": "object",
-                "additionalProperties": false,
-                "properties": {
-                    "application/json": {
-                        "type": "object"
-                    }
+                "additionalProperties": {
+                  "description": "mimetype output (e.g. text/plain), represented as either an array of strings or a string.",
+                  "$ref": "#/definitions/misc/multiline_string"
                 },
                 "patternProperties": {
-                    "^(?!application/json$)[a-zA-Z0-9]+/[a-zA-Z0-9\\-\\+\\.]+$": {
-                        "description": "mimetype output (e.g. text/plain), represented as either an array of strings or a string.",
-                        "$ref": "#/definitions/misc/multiline_string"
+                    "^application/([a-zA-Z0-9.]+\\+)?json$": {
+                        "description": "Mimetypes with JSON output, can be an object",
+                        "type": "object"
```

Comment:
I am getting a validation error when trying to save a notebook with output of "application/json" mime type and array data type. I am going to create a issue/PR for this...